lib/tasks/widgets/tasks_screen.dart

```diff
@@ -107,6 +120,19 @@ class _TasksScreenState extends State<TasksScreen> {
                           )
                         ],
                       ),
+                      //bla
+                      onTap: () => Navigator.of(context)
```

Comment:
This should probably be `onLongPress` to avoid accidentally editing tasks and provide a consistent interaction model with the tasks with subtasks.

Reply:
Once again, please also remove superfluous comments.